### PR TITLE
OWLS-89562 - Add option to create-domain.sh in FMW domain home in image sample for use in imagetool --buildNetwork parameter

### PR DESCRIPTION
**Pull request by alai8**
- Add -b option to create-domain.sh in FMW domain home in image sample for passing value into -buildNetwork parameter of imagetool.sh:

usage: create-domain.sh -o dir -i file -u username -p password [-q rcuSchemaPassword] [-b buildNetworkParam] [-n encryption-key] [-e] [-v] [-h]

- Update ItFmwDiiSample to use this new option

- Update FMW domain home in image samples doc and add troubleshooting section

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/5108/

### CODE REVIEW

**Review by sankarpn — APPROVED** [review]
LGTM

**Review by rjeberhard — APPROVED** [review]

File: documentation/staging/content/samples/simple/domains/fmw-domain-home-in-image/_index.md [hunk 1]
```diff
@@ -531,3 +532,37 @@ By default, they are installed under `/tmp/dhii-sample/tools` directory.
 ```shell
 $ rm -rf /tmp/dhii-sample/tools/
 ```
+### Troubleshooting
+***Message***: `Failed to build JDBC Connection object`
+
+If the WebLogic Image Tool failed to create a domain, and the following error is seen in the output:
```
Comment (rosemarymarano) [inline documentation/staging/content/samples/simple/domains/fmw-domain-home-in-image/_index.md:538]: domain, -> domain (no comma)

File: documentation/staging/content/samples/simple/domains/fmw-domain-home-in-image/_index.md [hunk 2]
```diff
@@ -531,3 +532,37 @@ By default, they are installed under `/tmp/dhii-sample/tools` directory.
...
+at com.oracle.cie.domain.script.jython.WLScriptContext.runCmd(WLScriptContext.java:738)
+at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
+at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
+at java.lang.reflect.Method.invoke(Method.java:498)
+
+com.oracle.cie.domain.script.jython.WLSTException: com.oracle.cie.domain.script.jython.WLSTException: Got exception when auto configuring the schema component(s) with data obtained from shadow table:
+Failed to build JDBC Connection object:
+```
+
+First verify that the JDBC connection URL shown in the output is correct. Update the `rcuDatabaseURL` parameter in the inputs YAML to the correct value if necessary.
```
Comment (rosemarymarano) [inline documentation/staging/content/samples/simple/domains/fmw-domain-home-in-image/_index.md:563]: First -> First, (comma)
inputs YAML -> inputs YAML file

File: documentation/staging/content/samples/simple/domains/fmw-domain-home-in-image/_index.md [hunk 3]
```diff
@@ -531,3 +532,37 @@ By default, they are installed under `/tmp/dhii-sample/tools` directory.
...
+
+If the JDBC connection URL  is correct, it is possible that the container in which the WebLogic Image Tool is running for creating a WebLogic domain is not using the correct networking stack.
```
Comment (rosemarymarano) [inline documentation/staging/content/samples/simple/domains/fmw-domain-home-in-image/_index.md:565]: extra space between URL  is
domain -> domain, (comma)

File: documentation/staging/content/samples/simple/domains/fmw-domain-home-in-image/_index.md [hunk 4]
```diff
@@ -531,3 +532,37 @@ By default, they are installed under `/tmp/dhii-sample/tools` directory.
...
+The optional `-b` option in the `create-domain.sh` can be used to specify the networking mode for the RUN instruction during image build.
```
Comment (rosemarymarano) [inline documentation/staging/content/samples/simple/domains/fmw-domain-home-in-image/_index.md:566]: in the `create-domain.sh` -> in the `create-domain.sh` script

**Review by rosemarymarano — COMMENTED** [review]
few minor edits

**Comment by alai8** [comment]
> few minor edits

Thanks for the review. Updated with suggested edits.

**Pull request merged**